### PR TITLE
feat(FX-4479): empty state for recent price ranges

### PR DESCRIPTION
**Pull request by dimatretyak**
This PR resolves [FX-4479] 

#### When the collector profile-sourced price range is NOT specified

https://user-images.githubusercontent.com/3513494/204297032-a5b6eeb9-5a14-4b47-a10c-dbd02c93b1aa.mp4

#### When the collector profile-sourced price range is specified
https://user-images.githubusercontent.com/3513494/204296983-316f39cc-c666-48c9-9825-090347630985.mp4

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [x] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [x] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Empty state for recent price ranges - dimatretyak

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[FX-4479]: https://artsyproduct.atlassian.net/browse/FX-4479?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Components/ArtworkFilter/RecentPriceRanges.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    return null
+  }
+
+  return (
+    <>
+      <Flex mx={2} flexDirection="row" justifyContent="space-between" alignItems="center">
+        <Text variant="sm">Recent price ranges</Text>
+
+        {!isEmptyPriceRanges && !!hasCustomPriceRange && (
+          <TouchableOpacity
```
Comment (gkartalis) [inline src/app/Components/ArtworkFilter/RecentPriceRanges.tsx:54]: Could you also add a11y role and label on this?
```suggestion
          <TouchableOpacity
              accessibilityRole="button"
              accessibilityLabel="Clear price ranges"
```

**Review by gkartalis — APPROVED** [review]

**Comment by gkartalis** [comment]
great job! Thanks for the test cases and videos! Added one small comment about accessibility other than that looks great!

File: src/app/Components/ArtworkFilter/RecentPriceRanges.tsx (hunk 1 above)
Comment (dimatretyak) [reply src/app/Components/ArtworkFilter/RecentPriceRanges.tsx:54]: yep, sure

Comment (dimatretyak) [reply src/app/Components/ArtworkFilter/RecentPriceRanges.tsx:54]: done

**Review by gkartalis — APPROVED** [review]

**Pull request merged**